docker/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.3'
+
+services:
+   db:
+     image: mysql:5.7
+     volumes:
+       - db_data:/var/lib/mysql
+     restart: always
+     environment:
+       MYSQL_ROOT_PASSWORD: m4ster_s3cret
+       MYSQL_DATABASE: libreehr
+       MYSQL_USER: libreehr
+       MYSQL_PASSWORD: s3cret
+   librehealth_ehr:
+     depends_on:
+       - db
+     image: librehealth_ehr:${PHP_DOCKER_VERSION}-apache
+     ports:
+       - "8000:80"
+     restart: always
+     environment:
+       LIBREHEALTH_DB_HOST: db
+       LIBREHEALTH_DB_PORT: 3306
+       LIBREHEALTH_DB_NAME: libreehr
+       LIBREHEALTH_DB_USER: libreehr
+       LIBREHEALTH_DB_PASSWORD: s3cret
+volumes:
+    db_data:
```

Comment:
```yaml
version: '3.3'

services:
   db:
     image: mysql:5.7
     volumes:
       - db_data:/var/lib/mysql
     restart: always
     environment:
       MYSQL_ROOT_PASSWORD: m4ster_s3cret
       MYSQL_DATABASE: libreehr
       MYSQL_USER: libreehr
       MYSQL_PASSWORD: s3cret
   librehealth_ehr:
     depends_on:
       - db
     # image: librehealth_ehr:${PHP_DOCKER_VERSION-php7.1}-apache
     content: ..
     ports:
       - "8000:80"
     restart: always
     environment:
       LIBREHEALTH_DB_HOST: db
       LIBREHEALTH_DB_PORT: 3306
       LIBREHEALTH_DB_NAME: libreehr
       LIBREHEALTH_DB_USER: libreehr
       LIBREHEALTH_DB_PASSWORD: s3cret
volumes:
    db_data:
```

Let's use this for now -- until we push the image. Let the image be built from the Dockerfile.